spec/features/user_updates_profile_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+# Acceptence Criteria
+# [√] I can navigate to an update page for my profile
+# [√] My profile is populated with accurate information
+
+feature "As an authenticated user
+I want to update my information
+So that I can keep my profile up to date" do
+  scenario "authenticated user clinks link to update information" do
+    user = FactoryGirl.create(:user)
+    email_update = 'newuseremail@email.com'
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.